app/internal/logger_customizer.py

```diff
@@ -0,0 +1,110 @@
+import json
+import sys
+from typing import Union
+
+from pathlib import Path
+from loguru import logger, _Logger as Logger
+
+
+class LoggerConfigError(Exception):
+    pass
+
+
+class LoggerCustomizer:
+
+    @classmethod
+    def make_logger(cls, config_file_or_dict: Union[Path, dict],
+                    logger_name: str) -> Logger:
+        """Creates a loguru logger from given configuration path or dict.
+
+        Args:
+            config_file_or_dict (Union[Path, dict]): Path to logger
+            configuration file or dictionary of configuration
+            logger_name (str): Logger instance created from configuration
+
+        Raises:
+            LoggerConfigError: Error raised when the configuration is invalid
+
+        Returns:
+            Logger: Loguru logger instance
+        """
+        config = cls.load_logging_config(config_file_or_dict)
+
+        try:
+            logging_config = config.get(logger_name)
+            logs_path = logging_config.get('path')
+            log_file_path = logging_config.get('filename')
+
+            logger = cls.customize_logging(
+                file_path=Path(logs_path) / Path(log_file_path),
+                level=logging_config.get('level'),
+                retention=logging_config.get('retention'),
+                rotation=logging_config.get('rotation'),
+                format=logging_config.get('format')
+            )
+        except (TypeError, ValueError) as err:
+            raise LoggerConfigError(
+                f"You have an issue in the logger configuration: {err!r}, "
```

Comment:
"issue with".

Reply:
I wanted to show the real error that's been caught, adding the !r gives us the __repr__ of the error and separates it from the rest of the text

Reply:
It's giving me the __repr__ of the error, leaving "" and helping separate the raw error from the rest of the text.